src/M6Web/Bundle/RedisBundle/DependencyInjection/M6WebRedisExtension.php

```diff
@@ -136,25 +134,38 @@ protected function loadClient($container, $alias, array $config, array $servers,
             throw new InvalidConfigurationException(sprintf("no server found for M6Redis client %s", $alias));
         }
 
+        $hashFunction = null;
+        if (array_key_exists('hash_function', $config) &&  $config['hash_function'] !== null) {
+            $hashFunction = new $config['hash_function'];
```

Comment:
Could we inject a service? May be very powerful if we have "complex" strategy to instanciate 😕